src/sophys/common/plans/parallel_wrapper.py

```diff
@@ -0,0 +1,44 @@
+from threading import Event, Thread
+
+
+def yield_msg(msg):
+    yield msg
+
+def all_plans_have_finished(plan_has_finished_list):
```

Comment:
Is this function's name right? It seems to be computing `not_all_plans_have_finished` instead.

Reply:
I inverted the logic when I was creating this function. Already changed the name.